packages/core/admin/server/src/bootstrap.ts

```diff
@@ -69,6 +69,43 @@ const syncAPITokensPermissions = async () => {
   }
 };
 
+/**
+ * Ensures the creation of default API tokens during the app creation.
+ *
+ * Checks the database for existing users and API tokens:
+ * - If there are no users and no API tokens, it creates two default API tokens:
+ *   1. A "Read Only" API token with permissions for accessing resources.
+ *   2. A "Full Access" API token with permissions for accessing and modifying resources.
+ *
+ * @sideEffects Creates new API tokens in the database if conditions are met.
+ */
+
+const createDefaultAPITokensIfNeeded = async () => {
+  const userService = getService('user');
+  const apiTokenService = getService('api-token');
+
+  const usersCount = await userService.count();
+  const apiTokenCount = await apiTokenService.count();
+
+  if (usersCount === 0 && apiTokenCount === 0) {
+    await apiTokenService.create({
+      name: 'Read Only',
+      description:
+        'A default API token with read-only permissions, only used for accessing resources',
```

Comment:
Not necessarily in this PR, but ideally we should look up a string in the default locale so this can be translated, otherwise non-english users might have very little idea what this is or why it's here.